engine/build.gradle

```
@@ -33,52 +33,58 @@ buildscript {
 }
 
 dependencies {
-    compile "com.badlogicgames.gdx:gdx:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-box2d:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-controllers:$gdxVersion"
+    api group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.25'
+    api group: 'com.google.code.gson', name: 'gson', version: '2.6.2'
+
+    api "com.badlogicgames.gdx:gdx:$gdxVersion"
+    api "com.badlogicgames.gdx:gdx-box2d:$gdxVersion"
+    api "com.badlogicgames.gdx:gdx-controllers:$gdxVersion"
+
+    api(group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '7.0.6-SNAPSHOT')
+    api(group: 'org.terasology.gestalt', name: 'gestalt-entity-system', version: '7.0.6-SNAPSHOT')
+    api(group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.0.6-SNAPSHOT')
+    api(group: 'org.terasology.gestalt', name: 'gestalt-util', version: '7.0.6-SNAPSHOT')
 
-    compile(group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '7.0.6-SNAPSHOT')
-    compile(group: 'org.terasology.gestalt', name: 'gestalt-entity-system', version: '7.0.6-SNAPSHOT')
-    compile(group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.0.6-SNAPSHOT')
-    compile(group: 'org.terasology.gestalt', name: 'gestalt-util', version: '7.0.6-SNAPSHOT')
-    compile group: 'org.terasology.nui', name: 'nui', version: '2.0.0'
-    compile group: 'org.terasology.nui', name: 'nui-libgdx', version: '2.0.0'
+    api group: 'org.terasology.nui', name: 'nui', version: '2.0.0'
+    api group: 'org.terasology.nui', name: 'nui-libgdx', version: '2.0.0'
 
-    compile group: 'com.google.protobuf', name: 'protobuf-java', version: '3.4.0'
+    implementation group: 'com.google.protobuf', name: 'protobuf-java', version: '3.4.0'
 
-    compile "com.github.everit-org.json-schema:org.everit.json.schema:1.11.1"
-    compile "com.github.marschall:zipfilesystem-standalone:1.0.1"
-    compile 'dom4j:dom4j:1.6.1'
+    implementation "com.github.zafarkhaja:java-semver:0.9.0" // gestalt lost this...
```

This is different to the JSemver version that gestalt uses. Gestalt uses JSemver `0.10.0` from the `heisluft.de` repository.
```suggestion
    implementation "com.github.zafarkhaja:java-semver:0.10.0" // gestalt lost this...
```
https://github.com/MovingBlocks/gestalt/blob/fc5e69a197eb37456284fb80ea1dd7d53b3d410c/gestalt-module/build.gradle#L43